src/Microsoft.AspNet.Razor/Parser/HtmlMarkupParser.Block.cs

```diff
@@ -471,11 +471,21 @@ private void BeforeAttribute()
                 return;
             }
 
+            // Minimized attribute
+            // Need to modify this code to account for space inbetween equals when
+            // https://github.com/aspnet/Razor/issues/123 is completed.
             if (!At(HtmlSymbolType.Equals))
             {
-                // Saw a space or newline after the name, so just skip this attribute and continue around the loop
-                Accept(whitespace);
-                Accept(name);
+                // Output anything prior to the attribute, in most cases this will be the tag name:
+                // |<input| checked />
```

Comment:
- This change remains confusing.  Very little else but whitespace has changed in this class.  So why does the same condition mean `// Minimized attribute` now but `// Saw a space or newline after the name` before?
- If `// Minimized attribute` is correct, the comment is in the wrong place -- belongs inside the block.  Don't place definitive comments before the check that makes them true.
- `in most cases` must be untrue unless minimized attributes are not supported after other (minimized or not) attributes.  Leave comment out or mention other, equally-likely possibilities.